README.md

```diff
@@ -111,6 +111,12 @@ user named `daemon` defined within that file-system.
             "source": "sysfs",
             "destination": "/sys",
             "options": "nosuid,noexec,nodev"
+        },
+        {
+            "type": "cgroup",
```

Comment:
Yeah, that's because we use bind mount, and I don't think that part of details should be exposed to users.